src/armasters.py

```diff
@@ -34,3 +46,107 @@ def __init__(self, ndet):
         self._msbias_name = [None for all in xrange(ndet)]     # Master Bias Name
```

Comment:
Just as a heads up... my intention for this class was to store all unique master frames. Then in the sciexp class (i.e. in arsciexp), arrays such as `self._msbias` store the index where the unique master frame is stored in the corresponding array in the MasterFrames class. At the moment, we store the actual data frames in the sciexp class. If multiple science frames use the same calibration data, we end up duplicating the same frame many times. This is expensive to store, especially while reducing a large amount of data. Storing an index, however, and just the single unique master calibration frame, is cheap. This comment is not to encourage you to make these (fairly substantial) changes now, it's just an FYI for future development.